### ISSUE TITLE
Rating Indicator: There is no alternative text provide for icons

### ISSUE THREAD

**Issue by puru-hk** [SAP/fundamental-ngx#5033]
Description: There is no alternative text provide for icons

Expected: Alternative text should be provided for all the icons

Screenshot:

![image](https://user-images.githubusercontent.com/32538291/112795198-ca0f9100-9085-11eb-992b-6489638b88c4.png)

**Comment by droshev** [comment]
@puru-hk there is an input `aria-label` and `name` on the component level which is used for each icon
![Screen Shot 2021-03-29 at 10 31 45 AM](https://user-images.githubusercontent.com/4380815/112852535-06d58b00-907a-11eb-8d25-624665410a23.png)
